components/WorkerView/WorkerRecap.tsx

```diff
@@ -1,21 +1,23 @@
 import { Worker } from 'types';
 
+import style from './WorkerRecap.module.scss';
+
 const WorkerRecap = ({
   firstName,
   lastName,
   role,
   teams,
 }: Worker): React.ReactElement => {
   return (
-    <div>
-      <h2 className="gov-weight-lighter">
+    <div className={style.detailsContainer}>
+      <h2 className={`lbh-heading-h2 gov-weight-lighter ${style.title}`}>
         Worker&apos;s details found in system
       </h2>
-      <p className="govuk-body">
+      <p className="lbh-body govuk-body">
```

Comment:
you don't need the extra govuk-body class for paras. in fact, you might not need any classes at all since i think we're using global styles